ulam-vs-random.py

```diff
@@ -1,19 +1,30 @@
 #!/usr/bin/env python3
 import random
 
-
+"""Seives Algo to find prime numbers within n . More Time Efficient. """
```

Comment:
The docstrings go inside the function, not outside it. They also follow certain conventions like imperative mood, single subject line, etc. See the following URLs for conventions used while writing Python code:

- https://www.python.org/dev/peps/pep-0008/
- https://www.python.org/dev/peps/pep-0257/